payments-ui-core/src/main/java/com/stripe/android/ui/core/forms/resources/LpmRepository.kt

```diff
@@ -110,78 +113,60 @@ class LpmRepository constructor(
      */
     @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
     fun update(
-        expectedLpms: List<String>,
-        serverLpmSpecs: String?
-    ) = internalUpdate(expectedLpms, serverLpmSpecs, false)
-
-    @VisibleForTesting
-    fun forceUpdate(
-        expectedLpms: List<String>,
-        serverLpmSpecs: String?
-    ) = internalUpdate(expectedLpms, serverLpmSpecs, true)
-
-    /**
-     * Will add the server specs to the repository and load the ones from disk if
-     * server is not parseable.
-     */
-    private fun internalUpdate(
-        expectedLpms: List<String>,
+        stripeIntent: StripeIntent,
         serverLpmSpecs: String?,
-        force: Boolean = false
     ) {
-        val newSpecsToLoad =
-            expectedLpms.firstOrNull { !lpmInitialFormData.containsKey(it) } != null
-
-        if (!isLoaded() || force || newSpecsToLoad) {
-            serverSpecLoadingState = ServerSpecState.NoServerSpec(serverLpmSpecs)
```

Comment:
Lots of whitespace changes starting here. Feel free to turn whitespaces changes off when viewing the diff:
https://github.com/stripe/stripe-android/pull/6011/files?diff=unified&w=1#diff-b7cb58f4f0edb84eed7ab0a30e0c25a8398661543060d1b6f83aef0ee5742523R116